enterprise/backend/src/metabase_enterprise/transforms/ordering.clj

```diff
@@ -125,9 +138,14 @@
         db-transforms    (filter #(= (get-in % [:source :query :database]) db-id) transforms)
         output-tables    (output-table-map mp db-transforms)
         transform-ids    (into #{} (map :id) db-transforms)
-        node->children   #(->> % transforms-by-id transforms.i/table-dependencies (keep (fn [{:keys [table transform]}]
-                                                                                          (or (output-tables table)
-                                                                                              (transform-ids transform)))))
+        target-refs      (target-ref-map transforms)
+        node->children   #(->> % transforms-by-id transforms.i/table-dependencies
+                               (keep (fn [{:keys [table transform table-ref]}]
```

Comment:
this whole `keep` is duplicated from above, may be nice to refactor in an utility function so there's not so much copy pasting